lib/rspec/matchers/expecteds_for_multiple_diffs.rb

```diff
@@ -46,8 +46,14 @@ def self.for_many_matchers(matchers)
       # @return [String]
       def message_with_diff(message, differ, actual)
         diff = diffs(differ, actual)
-        message = "#{message}\n#{diff}" unless diff.empty?
-        message
+        if diff == "Diff:\e[0m\n\e[0m"
```

Comment:
Prehaps this should actually be in `def diffs`

Reply:
yep completely. Sorry I was working on it on the train step by step but didn't push my last change in `diffs`. :)